src/arch/riscv/faults.cc

```diff
@@ -164,8 +164,6 @@ RiscvFault::invoke(ThreadContext *tc, const StaticInstPtr &inst)
         pc_state.set(addr);
         tc->pcState(pc_state);
     } else {
-        inst->advancePC(pc_state);
-        tc->pcState(pc_state);
```

Comment:
```suggestion
```
I think move the code section to `SyscallFault::invokeSE(ThreadContext *tc, const StaticInstPtr &inst)` will pass to test CI